src/cli/Cli.ts

```diff
@@ -787,11 +785,10 @@ export class Cli {
     }
   }
 
-  public static prompt(options: any, cb: (result: any) => void): void {
+  public static async prompt(options: any): Promise<{ continue: boolean }> {
```

Comment:
I think we might need to make it generic, because, if I'm not mistaken, we also use this function to prompt for missing values in the CLI runtime

Reply:
So something like this?

```typescript
public static async prompt<T>(options: any): Promise<T> {
    const inquirer: Inquirer = require('inquirer');
    const result = await inquirer.prompt(options);
    return result as T;
  }
```

Reply:
Yup, that would work. We might as well combine lines 3 and 4 to `return await inquirer.prompt(options) as T;`